code/modules/mob/living/carbon/shock.dm

```diff
@@ -4,7 +4,7 @@
 /mob/living/carbon/proc/adjustTraumatic_Shock(amount)
 	if(status_flags & GODMODE)
 		return FALSE	//godmode
-	traumatic_shock = clamp(traumatic_shock+amount,0,maxHealth*2)
+	traumatic_shock = clamp(traumatic_shock+amount,-100,maxHealth*2)
```

Comment:
why would traumatic shock be below 0?

Reply:
Painkillers. If you have 0 damage and you're on oxy and tramadol, your felt pain should be negative.